src/index.html

```diff
@@ -3,11 +3,312 @@
+              </div>
+              <div class="info__row">
+                <a href="#" class="info__title">Спогади і мрії</a>
+              </div>
+              <div class="info__row">
+                <div class="info__text subtitle">
+                  Національний художній музей України до 100 річчя від дня народження   видатної української художниці  Тетяни Яблонської  відкриває ретроспективну ювілейну виставку «І спогади і мрії».
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="lecture">
+        <div class="lecture__img"> </div>
+        <div class="container lecture__container">
+          <div class="lecture__info">
+            <div class="lecture__row lecture__top">
+              <div class="info__subtitle subtitle">ЛЕКЦІЯ</div>
+              <div class="info__date subtitle">26/08/2019 - 11:00</div>
+            </div>
+            <div class="lecture__row">
+              <div class="info__title info__title--no-pseudo">Від класицизму до романтизму</div>
+            </div>
+          </div>
+        </div>
+      </div>
+      </section>
+
+      <section class="gallery" id="gallery">
+        <div class="container gallery__container">
+          <div class="gallery__title title">Галерея</div>
+          <div class="swiper">
+            <!-- Additional required wrapper -->
+            <div class="swiper-wrapper">
+              <!-- Slides -->
+              <div class="swiper-slide">
+                <img class="swiper-img"
+                 src="images/pictures/gallery-1.svg"
+                  alt="gallery-1">
+              </div>
+              <div class="swiper-slide">
+                <img class="swiper-img"
+                src="images/pictures/gallery-2.svg"
+                alt="gallery-2">
+              </div>
+              <div class="swiper-slide">
+                <img class="swiper-img"
+                 src="images/pictures/gallery-1.svg"
+                  alt="gallery-1">
+              </div>
+              ...
+            </div>
+            <!-- If we need pagination -->
+            <div class="swiper-pagination"></div>
+
+            <!-- If we need navigation buttons -->
+
+            <!-- If we need scrollbar -->
+          </div>
+          <div class="gallery__grid">
+            <div class="grid__row">
+              <img
+              src="images/pictures/gallery-1280-1.svg"
+               alt="1"
+                class=" grid__img grid__img-1"
+                >
+              <img
+              src="images/pictures/gallery-1280-2.svg"
+                alt="2"
+                class="gird__img grid__img-2"
+                >
+            </div>
+            <div class="grid__row">
+              <img
+              src="images/pictures/gallery-1280-3.svg"
+                alt="1"
+                class="gird__img grid__img-3"
+                >
```

Comment:
You need to check and fix it everywhere. Choose more descriptive name for alt
```suggestion
              <img
                src="images/pictures/gallery-1280-3.svg"
                alt="1"
                class="gird__img grid__img-3"
              >
```